mods/beds/README.md

```diff
@@ -4,16 +4,16 @@ See license.txt for license information.
 
 Authors of source code
 ----------------------
-Originally by BlockMen (MIT)
+Originally by BlockMen (MIT)<br>
```

Comment:
Instead of sprinkling `<br>` everywhere (which is not nice for plaintext reading), I'd suggest just adding line breaks

Reply:
How would I do it other than using a space at the end?

Reply:
How about `\`

Reply:
<pre>
Authors of source code
----------------------
Originally by BlockMen (MIT)

Various Minetest developers and contributors (MIT)
</pre>

should render like this IIRC:


Authors of source code
----------------------
Originally by BlockMen (MIT)

Various Minetest developers and contributors (MIT)

Reply:
Using br or one space or `\`:
![test1](https://user-images.githubusercontent.com/49789044/75978172-dbfc2000-5ed5-11ea-8aa5-0f3e2c93d9e0.png)

Using an actual line break:
![test2](https://user-images.githubusercontent.com/49789044/75978175-dc94b680-5ed5-11ea-867a-d267a6d926df.png)

Which one do you prefer?